src/app/games/new-game/tests/NewGameHeuristic.spec.ts

```diff
@@ -19,8 +21,13 @@ describe('NewGameHeuristic', () => {
         /**
          * To test board values, most of the time you want to rely on
          * `HeuristicUtils.expectSecondStateToBeBetterThanFirstFor`.
+         * You can include last moves when needed (here there are set to MGPOptional.empty())
          */
-        const initialState: NewGameState = NewGameState.getInitialState();
-        HeuristicUtils.expectStatesToBeOfEqualValue(heuristic, initialState, initialState);
+        const weakState: NewGameState = NewGameState.getInitialState();
+        const strongState: NewGameState = new NewGameState(42);
+        HeuristicUtils.expectSecondStateToBeBetterThanFirstFor(heuristic,
```

Comment:
euh ? second the be better than first
first = strong
second = weak
what

Reply:
bien vu, fixé ! (le player était zero, mais vu que l'heuristique retourne le tour, c'est stronger pour ONE)